test/check/check.go

```diff
@@ -400,6 +439,16 @@ func checkForGenericDeployment(name, namespace string, checkLabels bool) error {
 	return nil
 }
 
+func checkWorkloadRelationLabels(labelMapList []map[string]string, kind, name string) error {
```

Comment:
question, what do you mean by `Workload`?

Reply:
https://cloud.google.com/kubernetes-engine/docs/how-to/deploying-workloads-overview
those that have template

Reply:
ah. understood. then it's a very good naming :) thanks!